src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/update/OverseasEntityUpdateSubmissionDtoValidatorTest.java

```diff
@@ -0,0 +1,201 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.update;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.gov.companieshouse.overseasentitiesapi.mocks.*;
+import uk.gov.companieshouse.overseasentitiesapi.model.BeneficialOwnersStatementType;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.*;
```

Comment:
need to remove the .* imports and replace with individual imports for each class